torch/csrc/distributed/c10d/init.cpp

```diff
@@ -1053,7 +1053,8 @@ that adds a prefix to each key inserted to the store.
   auto processGroupOptions =
       intrusive_ptr_class_<::c10d::ProcessGroup::Options>(
           module, "ProcessGroupOptions")
-          .def_readwrite("backend", &::c10d::ProcessGroup::Options::backend);
+          .def_readwrite("backend", &::c10d::ProcessGroup::Options::backend)
+          .def_readwrite("timeout", &::c10d::ProcessGroup::Options::timeout);
```

Comment:
We should document both the gloo options and nccl options including the arguments, defaults, and an example